### PR TITLE
Running PTP conformance test in a container

### PR DESCRIPTION
**Pull request by edcdavid**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -229,3 +229,7 @@ test-validation-only:
 # download controller-gen if necessary
 operator-sdk:
 	go install ./vendor/github.com/operator-framework/operator-sdk/cmd/operator-sdk
+buildtest:
+	PATH=${PATH}:${GOBIN} ginkgo build  ./test/conformance
```
Comment (aneeshkp) [inline Makefile:233]: can we build -o to  to bin folder

File: scripts/image.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+set -x
+VERSION=latest
+IMAGE_TAG=ptpconformance
+REPO=quay.io/deliedit
```
Comment (aneeshkp) [inline scripts/image.sh:5]: move  it to https://quay.io/repository/redhat-cne

File: scripts/runtestcontainer.sh [hunk 3]
```diff
@@ -0,0 +1,8 @@
+IMAGE=quay.io/deliedit/ptpconformance:latest
```
Comment (aneeshkp) [inline scripts/runtestcontainer.sh:1]: https://quay.io/repository/redhat-cne

File: test/README.md [hunk 4]
```diff
@@ -21,6 +21,15 @@ To run all the tests
 KUBECONFIG="/home/user/.kube/config" ENABLE_TEST_CASE=reboot PTP_TEST_MODE=Discovery make functests
```
Comment (aneeshkp) [inline test/README.md:21]: ~/usr/

File: test/README.md [hunk 5]
```diff
@@ -21,6 +21,15 @@ To run all the tests
...
 ```
 
+To run all the tests with a container:
```
Comment (aneeshkp) [inline test/README.md:24]: I would rather give  them plain  docker command so they don't have to checkout the project 
docker  run -e PTP_TEST_MODE=$TEST_MODE -e ENABLE_TEST_CASE=$ENABLE_TC -v $KUBECONFIG:/tmp/config:Z -v $OUTPUT:/output:Z  $IMAGE

Comment (edcdavid) [reply test/README.md:24]: fixed

File: Makefile (hunk 1 above)
Comment (edcdavid) [reply Makefile]: fixed

File: scripts/runtestcontainer.sh (hunk 3 above)
Comment (edcdavid) [reply scripts/runtestcontainer.sh]: fixed

File: test/README.md (hunk 4 above)
Comment (edcdavid) [reply test/README.md]: fixed

File: scripts/runtestcontainer.sh [hunk 6]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -x
+IMAGE=quay.io/redhat-cne/ptp-operator-test:latest
+KUBECONFIG=$1
+TEST_MODE=$2
+OUTPUT=$3
+ENABLE_TC=$4
+
+podman run -v $OUTPUT_LOC:/tmp:Z $TNF_IMAGE
```
Comment (aneeshkp) [inline scripts/runtestcontainer.sh:9]: Its upstream, so should be docker push

File: scripts/image.sh (hunk 2 above)
Comment (edcdavid) [reply scripts/image.sh]: done

File: scripts/runtestcontainer.sh (hunk 6 above)
Comment (edcdavid) [reply scripts/runtestcontainer.sh]: fixed

**Review by aneeshkp — APPROVED** [review]

**Pull request merged**